mediapipe/calculators/ovms/openvinomodelserversessioncalculator.cc

```diff
@@ -121,6 +156,9 @@ absl::Status OpenVINOModelServerSessionCalculator::GetContract(CalculatorContrac
     cc->OutputSidePackets().Tag(SESSION_TAG.c_str()).Set<std::shared_ptr<::InferenceAdapter>>();
     const auto& options = cc->Options<OpenVINOModelServerSessionCalculatorOptions>();
     RET_CHECK(!options.servable_name().empty());
+
+    OVMS_LOG_LEVEL = StringToLogLevel(std::string(std::getenv("OVMS_LOG_LEVEL") == nullptr ? "" : std::getenv("OVMS_LOG_LEVEL")));
```

Comment:
This should be option of a model server session not global variable. Why can't be just read FLAGS_minloglevel, as we do in OVMS repository, and perform full check in GetContract this way?